src/components/vote/proposals/OpenProposals.vue

```diff
@@ -341,11 +341,13 @@ export default class OpenProposals extends Vue {
       },
       {
         label: "Details",
+        key: "",
```

Comment:
All should have keys, no empty strings.

Reply:
@H34D is that cool to add a key value here or is that gonna break something?